pyiceberg/table/__init__.py

```diff
@@ -1624,6 +1638,19 @@ def to_ray(self) -> ray.data.dataset.Dataset:
 
         return ray.data.from_arrow(self.to_arrow())
 
+    def to_polars(self) -> pl.DataFrame:
+        """Read a Polars DataFrame from this Iceberg table.
+
+        Returns:
+            pl.DataFrame: Materialized Polars Dataframe from the Iceberg table
+        """
+        import polars as pl
+
+        result = pl.from_arrow(self.to_arrow())
+        if isinstance(result, pl.Series):
+            result = result.to_frame()
```

Comment:
nit should we be opinionated about this here? 
or just return the same signature as .to_arrow()
https://docs.pola.rs/api/python/dev/reference/api/polars.from_arrow.html#

Reply:
fair, no strong opinions here. Thanks!